src/views/RoomSelector.vue

```diff
@@ -106,18 +113,24 @@ export default {
 			rooms: [],
 			selectedRoom: null,
 			currentRoom: null,
+			searchText: '',
 			loading: true,
 		}
 	},
 	computed: {
 		availableRooms() {
-			return this.rooms.filter((room) => {
+			const roomsTemp = this.rooms.filter((room) => {
 				return room.type !== CONVERSATION.TYPE.CHANGELOG
 					&& (!this.currentRoom || this.currentRoom !== room.token)
 					&& (!this.showPostableOnly || room.readOnly === CONVERSATION.STATE.READ_WRITE)
 					&& room.objectType !== 'file'
 					&& room.objectType !== 'share:password'
 			})
+			if (!this.searchText) {
+				return roomsTemp
+			} else {
+				return roomsTemp.filter(x => x.name.includes(this.searchText))
```

Comment:
```suggestion
				return roomsTemp.filter(x => x.displayName.includes(this.searchText))
```

Should be display name otherwise it's the user id for one-to-one conversations

Reply:
Ok, thanks